src/backend/dx12/src/device.rs

```diff
@@ -6,8 +6,8 @@ use d3d12;
 use d3dcompiler;
 use dxguid;
 use kernel32;
-use spirv_cross::{hlsl, spirv, CompileTarget};
-use spirv_cross::spirv::ExecutionModel;
+use spirv_cross::hlsl::{self, ShaderModel};
```

Comment:
nit: can we just do `use spirv_cross::{hlsl, spirv}` instead?

Reply:
sure, done!